includes/class-alma-wc-payment-gateway.php

```diff
@@ -423,14 +423,61 @@ public function payment_fields() {
 					<?php if ( $n === $default_installments ) { ?>
 					checked
 					<?php	} ?>
+					onchange="
+						jQuery('.js-alma-payment-plan-table').hide();
+						jQuery('#alma-payment-plan-table-' + event.target.value + '-installments').show();
+					"
 				>
 				<label
 					class="checkbox"
 					for="alma_installments_count_<?php echo esc_attr( $n ); ?>"
 				>
 					<?php echo sprintf( esc_html__( '%d installments', 'alma-woocommerce-gateway' ), esc_html( $n ) ); ?>
 				</label>
-				<?php } ?>
+					<?php
+				}
+
+				$eligibilities = $this->get_cart_eligibilities();
+
+				if ( $eligibilities ) {
+					foreach ( $eligibilities as $n => $plan ) {
+						?>
```

Comment:
Tu peux peut-être sortir cette logique d'affichage dans une fonction pour mieux découper

Reply:
done in c1b66437bb767ae6bd0983812ac43daafe3a6e66 😉